zh_tw.lang

```diff
@@ -0,0 +1,209 @@
+gui.xaero_confirm =確認
+gui.xaero_cancel =取消
+gui.xaero_back =返回
+gui.xaero_delete =刪除
+gui.xaero_OK =確定
+gui.xaero_add_edit =添加/編輯
+gui.xaero_new_waypoint =新的路徑點
+gui.xaero_initial =初始化
+gui.xaero_edit_waypoint =編輯路徑點
+gui.xaero_armour_status_settings =盔甲狀態設置
+gui.xaero_choose_a_preset =預設方案
```

Comment:
Is this correct? It translates to "preset plan" with Google Translate, which isn't correct.

Reply:
This is correct in chinese statement. Which means to select from a bunch of presets.